### PR TITLE
Create-cash-register

### PR DESCRIPTION
**Pull request by FarwaMuhibZada**
# Cash_Register_App 🚀

## Project Description 📝

>
This project is a cash register application that calculates and displays change due to a customer based on the price of an item, the cash provided, and the cash in the register drawer. It provides different messages depending on the scenario, such as when exact change is provided, when the cash provided is insufficient, or when the register drawer cannot provide exact change.
that have this functionalities:
Input: Enter the amount of cash provided by the customer in the input field labeled "Enter Cash Provided".
Click: Click on the "Purchase" button to calculate the change due.
Output: The app will display messages in the #change-due element based on the calculated change or scenario.
Credits of This project is based on the FreeCodeCamp Cash Register project and serves as a simplified implementation.
>

## Demo 📸

life demo link(https://farwamuhibzada.github.io/Cash_Register_App/)
Capture

### CODE REVIEW

**Comment by MasomRezaie** [comment]
@FarwaMuhibZada, Dear Farwa You did a great job. Your project is well. your project has clean code, you used meaningful commends Everything in your project is perfect, but in your html file there shows an error, that you did not link html with css correctly. there should be styles.css, you wrote style.css.

File: index.html [hunk 1]
```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
```
Comment (FarwaMuhibZada) [inline index.html:6]: hello dear @MasomRezaie !
I check the code there was no problem  I linked the correct external css which is styles.css

**Comment by FarwaMuhibZada** [comment]
> @FarwaMuhibZada, Dear Farwa You did a great job. Your project is well. your project has clean code, you used meaningful commends Everything in your project is perfect, but in your html file there shows an error, that you did not link html with css correctly. there should be styles.css, you wrote style.css.

@MasomRezaie  thank you for reviewing and feedback .

File: script.js [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+};
+
+window.onload = displayCashInDrawer;
+purchaseBtn.addEventListener('click', checkRegister);
+
+cashInput.addEventListener('keydown', (e) => {
+  if (e.key === 'Enter') {
+    checkRegister();
+  }
+});
```
Comment (ahzamir) [inline script.js]: - [OPTIONAL] - Try to add your Event Listeners in while DOM content is loading, it helps to add all of your event listeners to be ready to the action right when your page loads and prevents any side effects below is an example to help you do that:

```javascript
document.addEventListener('DOMContentLoaded', () => {
  //In here add all of your addEventListener functions
  // don't forget to add the constants that you need in your event listeners here as the scope of this one is higher than other codes
  
  // Here add all of you code that belongs to event listeners
})
```

**Review by ahzamir — APPROVED** [review]
### Hi @FarwaMuhibZada 🙋‍♀️,

> Excellent work! 💯 You have done a great job implementing all the project requirements. 🥳 Your feature is complete! Congratulations!!!🎊

- **[REQUIRED]** Share your project on this [GROUP](https://www.facebook.com/groups/kick-startdev), add a simple Proud Moment text and a screenshot of your project then besides that add your project link at the end of the text as a GitHub Source code reference and post.

- Share your Project on your LinkedIn account and add a proud moment text with a screenshot.

- Well Done great design Implemented. 🎯🎉👍
image

# Status: Approved ✅
There is nothing to say other than to merge your branch. 🎉🎯✅👍😊🥳

WinkThumbsUpGIF

 ## Highlights 🌞

🎉 Great Design Implemented 
✔️ Created a professional Repo (You created a repository with your project name)
✔️ Correct use of Gitflow (You created the development branch and made it your default branch, then created and committed your work on your feature branch)
✔️ Commit messages are meaningful
✔️ Setup the linters correctly (You added the correct linters config into your project according to the language you used on your code)
✔️ No linter errors (All linters checks have been passed 🟢)
✔️ You have followed the Cash Register project task and committed using the command line
✔️ Created the pull request (From your feature branch to the development branch you created a pull request and submitted the pull request link for review)
✔️ Good PR description (You added a meaningful description to your PR about your work and the features you added on this pull request)
✔️Your code is clean
✔️Your repo has a professional README.md file
✔️ Your partner has reviewed your repo
✔️Your design is responsive

**Cheers 🥂 and Happy coding!!! 👯 Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Remember to tag me @ahzamir in your question so I can receive the notification.**

File: script.js (hunk 2 above)
Comment (FarwaMuhibZada) [reply script.js]: @ahzamir thanks for your valuable feedback. I applied  to my code

**Comment by ahzamir** [comment]
@FarwaMuhibZada 

Well done merge your branch

**Pull request merged**